package.json

```diff
@@ -9,15 +9,16 @@
     "type": "git",
     "url": "https://github.com/ableplayer/ableplayer.git"
   },
-  "main": "build/ableplayer.min.js",
+  "main": "build/ableplayer.dist.js",
   "types": "build/ableplayer.d.ts",
   "scripts": {
     "build": "grunt"
   },
   "dependencies": {
     "jquery": "^3.4.1",
     "js-cookie": "2.2.0",
-    "xml-js": "1.6.2"
+    "xml-js": "1.6.2",
+    "@vimeo/player": "2.10.0"
```

Comment:
best to have a caret ^ in front of the version. This means the package works with newer non breaking changes, like e.g. 2.11.7 (but not 3.x.x). That reduces the need in client projects to have many different versions of the same package.